### PR TITLE
docs:linux/sched.md

### PR DESCRIPTION
**Pull request by zdover23**
Rewrite sched.md into native-level
English.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

### CODE REVIEW

File: docs/linux/sched.md [hunk 1]
```diff
@@ -1,167 +1,173 @@
...
 
-3. When a write command completes, the zone write lock for the target zone of
-   the command is released and the dispatch process resumed. This means that if
-   the command at the head of the dispatch queue is targeting the same zone,
-   this command can now be issued (step 2.a).
+    3. When a write command completes, the zone write lock for the target zone
+       of the command is released and the dispatch process is resumed. This
+       means that if the command at the head of the dispatch queue targets the
+       same zone, then the command can be issued (when the write command
+       completes) (step 2.a).
```
Comment (damien-lemoal) [inline docs/linux/sched.md:64]: Nice rewrite. But the sub-clause of step (2) are numbered 1, 2 and 3, not a, b and c... So referencing 2.a is not working. Can you change the sub-bullets to be a, b and c ?

Comment (zdover23) [reply docs/linux/sched.md:64]: Good catch.

I've made this alteration.

**Comment by damien-lemoal** [comment]
Manually applied: I changed the ordered lists second level to html to get a correct rendering of the lettered sub-list items (Cinder does not render this properly). Thanks !

**Comment by zdover23** [comment]
Another will be on the way to you shortly.

**Pull request closed without merging**